src/chat/channel.py

```
@@ -305,31 +308,32 @@ def printRaw(self, name, text, scroll_forced=False):
         '''
         Print an raw message in the chatArea of the channel
         '''
-        try:
-            color = self.lobby.client.players.getUserColor(name)
-                
-            # Play a ping sound
-            if self.private and name != self.lobby.client.login:
-                self.pingWindow()
-                
-            # scroll if close to the last line of the log
-            scroll_current = self.chatArea.verticalScrollBar().value()
-            scroll_needed = scroll_forced or ((self.chatArea.verticalScrollBar().maximum() - scroll_current) < 20)
-            
-            cursor = self.chatArea.textCursor()
-            cursor.movePosition(QtGui.QTextCursor.End)
-            self.chatArea.setTextCursor(cursor)
-                                
-            formatter = Formatters.FORMATTER_RAW
-            line = formatter.format(time=self.timestamp(), name=name, color=color, width=self.maxChatterWidth, text=text)
-            self.chatArea.insertHtml(line)
-            
-            if scroll_needed:
-                self.chatArea.verticalScrollBar().setValue(self.chatArea.verticalScrollBar().maximum())
-            else:
-                self.chatArea.verticalScrollBar().setValue(scroll_current)
-        except:
-            pass
```

`except: pass`

Whaaaaaaat.